packages/cli/README.md

```diff
@@ -122,6 +123,22 @@ OPTIONS
 
 _See code: [@oclif/plugin-help](https://github.com/oclif/plugin-help/blob/v3.2.2/src/commands/help.ts)_
 
+## `coveo org:config:monitor [FILE]`
```

Comment:
I think this should be regenerated

Reply:
Mh, I was think about that: Should we document unreleased commands?

I would let the CI update the doc tbh. Like this we avoid playing ball with l23 and also ensure that the doc is updated whenever we release the command.